yabause/src/port/qt/ui/UIYabause.cpp

```diff
@@ -1169,6 +1170,25 @@ void UIYabause::on_aViewDebugMemoryEditor_triggered()
 	UIMemoryEditor( UIDebugCPU::PROC_MSH2, mYabauseThread, this ).exec();
 }
 
+void UIYabause::on_aProfilerToggle_triggered() {
+	YabauseLocker locker( mYabauseThread );
+   if (mYabauseThread->init() == 0 && MSH2 && SSH2) {
```

Comment:
Might handle caartdev detection here or only enable the option widget when dev cart has been selected.